typing-react-state/Exercise1/problem/src/App.tsx

```diff
@@ -1,16 +1,21 @@
-// Add/Correct the types and get rid of TypeScript errors
-// Things to take care of -
-//  - companyDescription should be allowed to be set to null
-//  - The "Reset Company Details" button has a setCompanyDetails call. Something here
-//    will have to be changed after you have added the correct type to the
-//    useState hook for companyDetails.
+/*
 
-import React, { useState, useEffect } from "react";
+Exercise:
+
+1) Fix the `companyDescription` state to allow a `null` value.
```

Comment:
I think this may need to be: `to allow strings or null`. Possibly? I read it and was thrown off because the companyDescription already allows that. Thoughts?

Reply:
Good call. I changed it to:

> 1) Fix the `companyDescription` state to allow a string or null value.